jyotisha/panchaanga/temporal/festival/applier/ecliptic.py

```diff
@@ -114,14 +115,31 @@ def assign_tropical_sankranti(self):
       10: "",
       11: "/ziziraRtuH/uttarAyaNam",
       12: "",
-    }
-
+      }
+    else:
+      RTU_MASA_TAGS = {
+        1: "/vasantaRtuH",
+        2: "",
+        3: "/grISmaRtuH",
+        4: "/dakSiNAyanam",
+        5: "/varSaRtuH",
+        6: "",
+        7: "/zaradRtuH",
+        8: "",
+        9: "/hEmantaRtuH",
+        10: "/uttarAyaNam",
+        11: "/ziziraRtuH",
+        12: "",
+      }
     for d in range(self.panchaanga.duration_prior_padding, self.panchaanga.duration + self.panchaanga.duration_prior_padding):
       if self.daily_panchaangas[d].tropical_date_sunset.month_transition is not None:
         jd_transition = self.daily_panchaangas[d].tropical_date_sunset.month_transition
 
         # Addsankranti
-        masa_id = (self.daily_panchaangas[d + 1].tropical_date_sunset.month - 1) % 12 + 1
+        if self.computation_system.tropical_month_start == 'madhava_at_equinox':
+          masa_id = (self.daily_panchaangas[d + 1].tropical_date_sunset.month - 1) % 12 + 1
```

Comment:
Try replacing this with 
masa_id = self.daily_panchaangas[d + 1].tropical_date_sunset.month 

:-D

Reply:
I thought the IDs are expected to be in 1, 2, ..., 12.

Reply:
I suspect that self.daily_panchaangas[d + 1].tropical_date_sunset.month has the same range ..

Reply:
The error seems here: `self.computation_system.tropical_month_start` - It should be `self.panchaanga.computation_system.tropical_month_start` of course. But even that doesn't fix. `computation_system` seems to be `MULTI_NEW_MOON_SIDEREAL_MONTH_ADHIKA__CHITRA_AT_180`. I am surely missing something